src/services/api/restBuilder/restBuilder.ts

```diff
@@ -54,7 +54,7 @@ export default function restBuilder<Data extends BaseData>(
 } {
   function useRestList(query?: ListQuery<Data>): Data[] {
     const { data, error } = useSWR<Data[], Error>(
-      path,
+      [path, JSON.stringify(query)],
```

Comment:
This causes `identifier` to be undefined down below for a delete and edit request, this causes the error message to print out the objects id instead of it's name. We need another way to display the name for our toast messages on delete and edit with this since it's no longer cached for these queries.

(this is also causing a few delete/edit tests to fail)